devel/libeclxx/include/ert/ecl/EclKW.hpp

```diff
@@ -77,6 +77,11 @@ namespace ERT {
             return *static_cast< T* >( ecl_kw_iget_ptr( this->m_kw, i ) );
         }
 
+        const typename std::remove_pointer< T >::type* data() const {
```

Comment:
This I do not understand; does this trickery by some magic give you *value* semantics with a copy? Or is this just a compilcated way of writing something like:
```C++
const T * data() const {
    return static_cast<const T*>( ecl_kw_get_ptr( this->m_kw ));
}
```